rootfs/etc/nginx/lua/balancer/sticky.lua

```diff
@@ -4,24 +4,26 @@ local util = require("util")
 local ck = require("resty.cookie")
 
 local _M = balancer_resty:new({ factory = resty_chash, name = "sticky" })
+local default_cookie_name = "route"
```

Comment:
The convention is to use uppercase for constants.

Reply:
fixed